"Informer not found for namespace" should not pop up when using hub mode

In hub mode, the cis only watch on specific hub namespace, the application namespace not has a Informer is as expect.

With the CIS arguments as below
~~~
          args: [
            "--bigip-username=$(BIGIP_USERNAME)",
            "--bigip-password=$(BIGIP_PASSWORD)",
            "--log-level=DEBUG",
            "--manage-ingress=false",
            "--node-poll-interval=60",
            "--periodic-sync-interval=36000",
            "--manage-configmaps=true",
            "--bigip-url=10.1.10.240",
            "--insecure=true",
            "--hubmode=true",
            "--filter-tenants=true",
            "--log-as3-response=true",
            "--bigip-partition=k8s",
            "--pool-member-type=cluster",
            "--namespace=f5-hub-1",
            "--namespace=f5-hub-2"
            ]
~~~

Note that only namespace _f5-hub-1_ and _f5-hub-2_ has informer created, the other namespace show not has informer, so the following kinds of log should not pop up.

~~~
2024/01/15 05:00:26 [ERROR] Informer not found for namespace: cistest002
2024/01/15 05:00:26 [ERROR] Informer not found for namespace: cistest003
2024/01/15 05:00:26 [ERROR] Informer not found for namespace: cistest001
~~~

--- a/pkg/appmanager/appManager.go
+++ b/pkg/appmanager/appManager.go
@@ -3065,7 +3065,9 @@ func (appMgr *Manager) getServicePortsFromEndpoint(ep *v1.Endpoints) map[int32]i
 	svcKey := fmt.Sprintf("%s/%s", ep.Namespace, ep.Name)
 	comInf, ok := appMgr.getNamespaceInformer(ep.Namespace)
 	if !ok {
-		log.Errorf("Informer not found for namespace: %v", ep.Namespace)
+		if !appMgr.hubMode {
+			log.Errorf("Informer not found for namespace: %v", ep.Namespace)
+		}
 		return nil
 	}
 	svc, exists, err = comInf.svcInformer.GetIndexer().GetByKey(svcKey)

